NewKernel_d/include/CGAL/NewKernel_d/LA_eigen/LA.h

```diff
@@ -115,12 +115,12 @@ template<class NT_,class Dim_,class Max_dim_=Dim_> struct LA_eigen {
 		return (int)v.cols();
 	}
 
-    template<class Mat_> static NT determinant_aux(Mat_ const& m, Tag_true) {
-        CGAL_assume(false);
-    }
-    template<class Mat_> static NT determinant_aux(Mat_ const& m, Tag_false) {
-        return m.determinant();
-    }
+	template<class Mat_> static CGAL_NORETURN NT determinant_aux(Mat_ const&, Tag_true) {
+		CGAL_error();
```

Comment:
For the principle, I don't like pessimizing code for the sake of a warning, even if the pessimization is negligible. Would it work to keep `CGAL_assume(false);` followed by `CGAL_error();` (or `return 0;`), or would you then get a warning that there is dead code?